gradle/compile-js-multiplatform.gradle

```diff
@@ -39,14 +47,25 @@ compileTestKotlinJs {
     kotlinOptions.metaInfo = true
     kotlinOptions.sourceMap = true
     kotlinOptions.moduleKind = 'umd'
+    kotlinOptions.freeCompilerArgs += "-XXLanguage:-NewInference"
+}
+
+if (project.tasks.findByName('compileKotlinJsIr')) {
+    compileKotlinJsIr {
+        kotlinOptions.freeCompilerArgs += "-XXLanguage:-NewInference"
```

Comment:
Please elaborate why NI is disabled

Reply:
Same question regarding atomicfu

Reply:
Reverted. It used to fail but now new inference is fixed for JS IR and disabled for classic backend.